Directory.Build.props

```diff
@@ -1,4 +1,5 @@
 ﻿<Project>
   <PropertyGroup>
+    <LangVersion>8.0</LangVersion>
```

Comment:
Problematic for the fsharp project? (as that's only `4.7`...)

I think `latest` and `preview` work for that too

Reply:
Good point. Pushed, let's see.